db/seeds.rb

```diff
@@ -0,0 +1,18 @@
+require 'active_record'
+require 'roo'
+require_relative '../app/models/verse'
+
+Load 100 verses from an .xlsx document into the Verses table
+xlsx_path = 'db/seed_data/100_verses.xlsx'
+xlsx = Roo::Spreadsheet.open(xlsx_path)
+sheet = xlsx.sheet(0)
+
+(2..101).each do |i|
+  notation = sheet.cell(i, 1)
+  verse_text = sheet.cell(i, 2)
+  Verse.create(scripture: verse_text, notation: notation, version: 'CSB', day: i - 1)
```

Comment:
⛏️ Shouldn't this version attribute come from the file, too?

Reply:
So for now the only verses they gave me were in CSB version. Eventually we will want to allow them to select which version they want to use. Maybe I should go ahead and add the version to the 3rd column so that way I don't have to fix it later?